### PR TITLE
use snapshot testing for clone

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: test/Cake.MinVer.Tests/Cake.MinVer.Tests.csproj [hunk 1]
```diff
@@ -8,8 +8,8 @@
   <ItemGroup>
     <PackageReference Include="Cake.Common" Version="1.0.0" />
     <PackageReference Include="Cake.Testing" Version="1.0.0" />
-
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.0" />
+    <PackageReference Include="Verify.Xunit" Version="11.0.0-beta.9" />
```
Comment (augustoproiete) [inline test/Cake.MinVer.Tests/Cake.MinVer.Tests.csproj:12]: Any interesting reason to use `11.0.0-beta.9` prerelease or would I be fine with stable `10.9.1`?

**Review by augustoproiete — APPROVED** [review]
LGTM! Thanks a lot @SimonCropp !!

File: test/Cake.MinVer.Tests/Cake.MinVer.Tests.csproj (hunk 1 above)
Comment (SimonCropp) [reply test/Cake.MinVer.Tests/Cake.MinVer.Tests.csproj:12]: you would be. i will be deploying a stable of 11 in the next 24hours. leave this open and i will move it to 11

Comment (augustoproiete) [reply test/Cake.MinVer.Tests/Cake.MinVer.Tests.csproj:12]: Awesome! Cheers mate!

**Comment by augustoproiete** [comment]
@SimonCropp your changes have been merged, thanks for your contribution :+1:

**Pull request merged**